content/docs/deployment/csmoperator/modules/observability.md

```diff
@@ -14,30 +14,40 @@ The CSM Observability module for supported Dell CSI Drivers can be installed via
   kubectl create namespace karavi
   ```
 - Enable Observability module and components in [sample manifests](https://github.com/dell/csm-operator/tree/main/samples). If cert-manager has already been installed, don't enable it.
-  - Scenario 1: Deploy one supported CSI Driver and enable Observability module
-    - If you enable `metrics-powerscale` or `metrics-powerflex`, must enable `otel-collector` as well.
-    
-  - Scenario 2: Deploy multiple supported CSI Drivers and enable Observability module
-    - When deploying the first driver, enable all components of Observability module in the CR. 
-    - For the following drivers, only enable the metrics service, and remove `topology` and `otel-collector` sections from the CR.
-    - The CR created at first must be deleted at last.
-    - 
-Note: pods in the `karavi` namespace will be in the ContainerCreating state until certificates are successfully created as described in the next step.
+- Observability will deploy with self-signed certificates by default. If you want to have custom certificates created instead, please generate certificates and private keys, encode them in base64, and insert them into the sample file as shown below for whichever components you are enabling:
```

Comment:
Is there an opportunity for the user to make a mistake in the generation and encoding? Any way to have the encoding validated prior to installation? Any useful log messages in case things go wrong?

Reply:
@donatwork if the encoding is in an improper format, there will be log messages in the operator pod indicating as such. Do you want some example logs here, or do I just tell the user to check the operator controller-manager logs?

Reply:
As long as the user knows how to check and the error is obvious. Adding too much error messages may become messy.

Reply:
what I can do is give the user tips on where in the logs to look when things go wrong

Reply:
@donatwork added debug info